src/semantic-ui/site/elements/image.overrides

```diff
@@ -22,3 +24,11 @@ Image Overrides - REACT-INVENIO-APP-ILS
       drop-shadow(0px -1px 0px #eee) drop-shadow(-1px 0px 0px #eee);
   }
 }
+
+.card-group-image-cover {
```

Comment:
I would suggest to use already existing class `fs-book-card` instead of creating new one